### PR TITLE
Relax pydantic constraint for client

### PR DESCRIPTION
**Pull request by dmchoiboi**
# Pull Request Summary

_What is this PR changing? Why is this change being made? Any caveats you'd like to highlight? Link any relevant documents, links, or screenshots here if applicable._

Upping pydantic requirement is proving problematic when rolling out. Will relax the requirement and autogenerate v1 compatible objects instead

Also, add client methods for getting and cancelling batch completions 

## Test Plan and Usage Guide

_How did you validate that your PR works correctly? How do you run or demo the code? Provide enough detail so a reviewer can reasonably reproduce the testing procedure. Paste example command line invocations if applicable._

### CODE REVIEW

**Comment by yunfeng-scale** [comment]
what's the error when rollout?

**Comment by dmchoiboi** [comment]
> what's the error when rollout?

Was testing the prior client out with some users and we ran into issues w/ some of their code still being on Pydantic v1. To facilitate migration, I decided to just update the client to also be pydantic v1 compatible, even though we really should move everyone off

**Review by yunfeng-scale — APPROVED** [review]

**Comment by yunfeng-scale** [comment]
> > what's the error when rollout?
> 
> Was testing the prior client out with some users and we ran into issues w/ some of their code still being on Pydantic v1. To facilitate migration, I decided to just update the client to also be pydantic v1 compatible, even though we really should move everyone off

how hard is it for them to migrate?

**Comment by dmchoiboi** [comment]
> how hard is it for them to migrate?

I don't know; but I don't want to put myself in a position where people aren't using this new API cause they're blocked on migrating

File: clients/python/llmengine/completion.py [hunk 1]
```diff
@@ -507,9 +509,15 @@ def batch_create(
             data_parallelism (int):
                 The number of parallel jobs to run. Data will be evenly distributed to the jobs. Defaults to 1.
 
+            priority (str):
```
Comment (Georgepu1) [inline clients/python/llmengine/completion.py:512]: nit: comment for `labels`

**Pull request merged**